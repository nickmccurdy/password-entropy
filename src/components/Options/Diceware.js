import { TextField } from 'material-ui'
import React from 'react'

import { dicewareWords } from '../PossiblePasswordsContainer'

export default function Diceware() {
  return <TextField disabled label="Possible Items" value={dicewareWords} />
}
