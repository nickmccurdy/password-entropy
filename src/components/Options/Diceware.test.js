import { shallow } from 'enzyme'

import Diceware from './Diceware'

test('Diceware', () => {
  expect(shallow(<Diceware />)).toMatchSnapshot()
})
