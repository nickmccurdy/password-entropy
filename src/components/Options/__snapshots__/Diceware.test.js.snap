// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Diceware 1`] = `
<TextField
  disabled={true}
  label="Possible Items"
  required={false}
  select={false}
  value={7776}
/>
`;
